### ISSUE TITLE
Prepare - fetch Ghost failing on ansible 2.2.0

### ISSUE THREAD

**Issue by rafalcymerys** [mtpereira/ansible-ghost#10]
I came across and error while running this playbook with ansible 2.2.

During "prepare - fetch Ghost" step I kept getting this error:

```
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/usr/local/Cellar/ansible/2.2.0.0_2/libexec/lib/python2.7/site-packages/ansible/executor/task_executor.py", line 119, in run
    res = self._execute()
  File "/usr/local/Cellar/ansible/2.2.0.0_2/libexec/lib/python2.7/site-packages/ansible/executor/task_executor.py", line 490, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/local/Cellar/ansible/2.2.0.0_2/libexec/lib/python2.7/site-packages/ansible/plugins/action/normal.py", line 33, in run
    results = merge_hash(results, self._execute_module(tmp=tmp, task_vars=task_vars))
  File "/usr/local/Cellar/ansible/2.2.0.0_2/libexec/lib/python2.7/site-packages/ansible/plugins/action/__init__.py", line 617, in _execute_module
    self._fixup_perms2(remote_files, remote_user)
  File "/usr/local/Cellar/ansible/2.2.0.0_2/libexec/lib/python2.7/site-packages/ansible/plugins/action/__init__.py", line 394, in _fixup_perms2
    ' see https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user'.format(res['rc'], res['stderr']))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 29: ordinal not in range(128)

fatal: [52.50.218.153]: FAILED! => {
    "failed": true, 
    "msg": "Unexpected failure during module execution.", 
    "stdout": ""
}
```

It seems like the newest version of ansible doesn't allow creating tmp files with certain permissions. I wonder if there's any reason to use 644 for the temporarily downloaded ghost zip file.

**Comment by mtpereira** [comment]
Hey!

Thanks for the report, I'll give it a look this week, possibly today.

**Comment by mtpereira** [comment]
Hey @rafalcymerys ,

So, has everything related to computers, with is complicated! Since this role depends on the [jdauphant.nginx](https://github.com/jdauphant/ansible-role-nginx) role and this other role is stuck on Ansible 2.1.0 due to [this other bug in Ansible](https://github.com/ansible/ansible/issues/17490), I don't feel very compelled to upgrade this to 2.2.0.0.

However, I did test this issue by disabling the nginx role on the tests and it runs fine on my machine using Ansible 2.2.0.0. Is there any special characteristic of your setup?

Thanks!

**Comment by mtpereira** [comment]
Fixed by PR #11.
